sdk/java-sdk/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -71,6 +71,7 @@ class DiscoveryImpl(system: ActorSystem, services: Map[String, Service]) extends
    */
   override def discover(in: ProxyInfo): scala.concurrent.Future[Spec] = {
     if (isVersionProbe(in)) {
+      log.info("this version is probe. The service info is: [{}]", serviceInfo)
```

Comment:
"this version is probe" is not something that users will understand. I don't understand it myself. :-)

I think we emit a log similar to what we are logging in the `else` branch of this `if`.